src/layers/MultiscaleImageLayer/MultiscaleImageLayer.js

```diff
@@ -170,7 +170,7 @@ export default class MultiscaleImageLayer extends CompositeLayer {
         modelMatrix: layerModelMatrix.scale(2 ** (numLevels - 1)),
         visible:
           opacity === 1 &&
-          -numLevels > this.context.viewport.zoom &&
+          -numLevels < this.context.viewport.zoom &&
             (!viewportId || this.context.viewport.id === viewportId),
```

Comment:
what is this last check?

Reply:
We only show the background image if the opacity is 1 (otherwise the tiled image will blend with the background image), the zoom level is high (i.e zoomed in), and we are in the correct viewport.